result.hash.temp

```diff
@@ -0,0 +1,37 @@
+{"utc_offset"=>-14400000,
```

Comment:
Put this here while adding new fields so I can see the structure of the data response from meetup api.